src/redux/counter/counterSlice.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Why you disabled this file ??

Reply:
in that case, ESLint thinks I'm reassigning or mutating parameter of the state 
and as it is referenced [HERE](https://redux.js.org/tutorials/essentials/part-2-app-structure#:~:text=important%20to%20remember%3A-,warning,state%20and%20cause%20bugs!,-With%20that%20in) using redux toolkit you are not really reassigning the state because everything is being handled within the redux toolkit itself. so that error was not valid